stdlib/etc/0/etc.rbs

```diff
@@ -0,0 +1,745 @@
+
+  SC_GETPW_R_SIZE_MAX: Integer
+
+  SC_HOST_NAME_MAX: Integer
+
+  SC_IOV_MAX: Integer
+
+  SC_IPV6: Integer
+
+  SC_JOB_CONTROL: Integer
+
+  SC_LINE_MAX: Integer
+
+  SC_LOGIN_NAME_MAX: Integer
+
+  SC_MAPPED_FILES: Integer
+
+  SC_MEMLOCK: Integer
+
+  SC_MEMLOCK_RANGE: Integer
+
+  SC_MEMORY_PROTECTION: Integer
+
+  SC_MESSAGE_PASSING: Integer
+
+  SC_MONOTONIC_CLOCK: Integer
+
+  SC_MQ_OPEN_MAX: Integer
+
+  SC_MQ_PRIO_MAX: Integer
+
+  SC_NGROUPS_MAX: Integer
+
+  SC_NPROCESSORS_CONF: Integer
+
+  SC_NPROCESSORS_ONLN: Integer
+
+  SC_OPEN_MAX: Integer
+
+  SC_PAGESIZE: Integer
+
+  SC_PAGE_SIZE: Integer
+
+  SC_PHYS_PAGES: Integer
+
+  SC_PRIORITIZED_IO: Integer
+
+  SC_PRIORITY_SCHEDULING: Integer
+
+  SC_RAW_SOCKETS: Integer
+
+  SC_READER_WRITER_LOCKS: Integer
+
+  SC_REALTIME_SIGNALS: Integer
+
+  SC_REGEXP: Integer
+
+  SC_RE_DUP_MAX: Integer
+
+  SC_RTSIG_MAX: Integer
+
+  SC_SAVED_IDS: Integer
+
+  SC_SEMAPHORES: Integer
+
+  SC_SEM_NSEMS_MAX: Integer
+
+  SC_SEM_VALUE_MAX: Integer
+
+  SC_SHARED_MEMORY_OBJECTS: Integer
+
+  SC_SHELL: Integer
+
+  SC_SIGQUEUE_MAX: Integer
+
+  SC_SPAWN: Integer
+
+  SC_SPIN_LOCKS: Integer
+
+  SC_SPORADIC_SERVER: Integer
+
+  SC_SS_REPL_MAX: Integer
+
+  SC_STREAM_MAX: Integer
+
+  SC_SYMLOOP_MAX: Integer
+
+  SC_SYNCHRONIZED_IO: Integer
+
+  SC_THREADS: Integer
+
+  SC_THREAD_ATTR_STACKADDR: Integer
+
+  SC_THREAD_ATTR_STACKSIZE: Integer
+
+  SC_THREAD_CPUTIME: Integer
+
+  SC_THREAD_DESTRUCTOR_ITERATIONS: Integer
+
+  SC_THREAD_KEYS_MAX: Integer
+
+  SC_THREAD_PRIORITY_SCHEDULING: Integer
+
+  SC_THREAD_PRIO_INHERIT: Integer
+
+  SC_THREAD_PRIO_PROTECT: Integer
+
+  SC_THREAD_PROCESS_SHARED: Integer
+
+  SC_THREAD_SAFE_FUNCTIONS: Integer
+
+  SC_THREAD_SPORADIC_SERVER: Integer
+
+  SC_THREAD_STACK_MIN: Integer
+
+  SC_THREAD_THREADS_MAX: Integer
+
+  SC_TIMEOUTS: Integer
+
+  SC_TIMERS: Integer
+
+  SC_TIMER_MAX: Integer
+
+  SC_TRACE: Integer
+
+  SC_TRACE_EVENT_FILTER: Integer
+
+  SC_TRACE_EVENT_NAME_MAX: Integer
+
+  SC_TRACE_INHERIT: Integer
+
+  SC_TRACE_LOG: Integer
+
+  SC_TRACE_NAME_MAX: Integer
+
+  SC_TRACE_SYS_MAX: Integer
+
+  SC_TRACE_USER_EVENT_MAX: Integer
+
+  SC_TTY_NAME_MAX: Integer
+
+  SC_TYPED_MEMORY_OBJECTS: Integer
+
+  SC_TZNAME_MAX: Integer
+
+  SC_V6_ILP32_OFF32: Integer
+
+  SC_V6_ILP32_OFFBIG: Integer
+
+  SC_V6_LP64_OFF64: Integer
+
+  SC_V6_LPBIG_OFFBIG: Integer
+
+  SC_VERSION: Integer
+
+  SC_XOPEN_CRYPT: Integer
+
+  SC_XOPEN_ENH_I18N: Integer
+
+  SC_XOPEN_REALTIME: Integer
+
+  SC_XOPEN_REALTIME_THREADS: Integer
+
+  SC_XOPEN_SHM: Integer
+
+  SC_XOPEN_STREAMS: Integer
+
+  SC_XOPEN_UNIX: Integer
+
+  SC_XOPEN_VERSION: Integer
+
+  VERSION: String
+
+  class Group < Struct[untyped]
+    extend Enumerable[untyped]
+
+    def self.[]: (*untyped) -> untyped
+
+    def self.each: () -> untyped
+
+    def self.inspect: () -> untyped
+
+    def self.keyword_init?: () -> untyped
+
+    def self.members: () -> untyped
+
+    def self.new: (*untyped) -> untyped
```

Comment:
[ask] These methods of `Struct` seems unnecessary to me. Is it better to remove them?

Reply:
I'm good to keep these methods. I don't think deleting them doesn't make a big impact to end users. even while it will make the type definition a bit more precise...